data/man/vala-language-server.1.scd

```diff
@@ -0,0 +1,29 @@
+vala-language-server(1)
+
+# NAME
+
+vala-language-server - Code Intelligence for Vala
+
+# SYNOPSIS
+
+*vala-language-server*
+
+# DESCRIPTION
+
+vala-language-server is a language server protocol implementation for Vala.
+It enables features like auto complete, go to defition or documentation
```

Comment:
Just some typos and other corrections:

    It enables features like auto-complete, go-to-definition, hover, symbol documentation, and more for editors/plugins supporting Vala and the Language Server Protocol.